CHANGELOG.md

```diff
@@ -70,6 +82,22 @@
 * Automatic analyze and optimize database [#4199](https://github.com/mixxxdj/mixxx/pull/4199)
 * Re-import and update metadata after files have been modified when loading tracks [#4218](https://github.com/mixxxdj/mixxx/pull/4218)
 * Fix playlists sidebar navigation/activation [#4193](https://github.com/mixxxdj/mixxx/pull/4193) [lp:1939082](https://bugs.launchpad.net/mixxx/+bug/1939082)
+* Sidebar: Left key jumps to parent and activates it [#4253](https://github.com/mixxxdj/mixxx/pull/4253)
+* Fix DEBUG ASSERT when navigating history playlists [#4297](https://github.com/mixxxdj/mixxx/pull/4297) [lp:1943643](https://bugs.launchpad.net/mixxx/+bug/1943643)
+* Library: add keyboard short to open track properties Ctrl+Enter [#4347](https://github.com/mixxxdj/mixxx/pull/4347)
+* Reanable short cut keys after editing controls
+  [#4360](https://github.com/mixxxdj/mixxx/pull/4360)
+  [lp:1902125](https://bugs.launchpad.net/mixxx/+bug/1902125)
+  [lp:1943325](https://bugs.launchpad.net/mixxx/+bug/1943325)
+* History feature: fix right-click index [#4384](https://github.com/mixxxdj/mixxx/pull/4384)
+* Allow to remove tracks by pressing Delete key
+  [#4330](https://github.com/mixxxdj/mixxx/pull/4330)
+  [lp:1227676](https://bugs.launchpad.net/mixxx/+bug/1227676)
+  [lp:1851457](https://bugs.launchpad.net/mixxx/+bug/1851457)
+  [lp:1856402](https://bugs.launchpad.net/mixxx/+bug/1856402)
+  [lp:1944565](https://bugs.launchpad.net/mixxx/+bug/1944565)
+* Keep selected track when browthing though the library, save model state [#4177](https://github.com/mixxxdj/mixxx/pull/4177)
+* Allow to remove a track form the disk [#3212](https://github.com/mixxxdj/mixxx/pull/3212)
```

Comment:
It's unclear if we should keep this feature, see discussion in #4560 
I'd appreciate if you take look at this issue and share your opinion there.

Reply:
If we keep it: "from"